src/styles/vocabulary_icons.scss

```diff
@@ -101,13 +107,25 @@ $linkedin-brand: #0077b5;
 $facebook-brand: #3b5998;
 $pinterest-brand: #bd081c;
 $slack-brand: #7c2852;
+$reddit-brand: #ff4500;
+$github-brand: #333333;
+$whatsapp-brand: #25d366;
+$tumblr-brand: #35465c;
+$sms-brand: #4aae20;
```

Comment:
Which color did you choose here?

Reply:
Ahh! Chose a color similar to whatsapp here 😅, since I was not sure what to use.